<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-P030MJJ" failures="1" tests="1" name="Test1" time="4.087" errors="0" timestamp="2024-08-17T00:07:58 IST">
  <testcase classname="day8RestAssuredAPIChaining.CreateUser" name="tes_createUser" time="4.043">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;io.restassured.internal.common.path.ObjectConverter.convertObjectTo(Object, java.lang.Class)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "io.restassured.internal.common.path.ObjectConverter.convertObjectTo(Object, java.lang.Class)" is null
at io.restassured.path.json.JsonPath.getInt(JsonPath.java:245)
at day8RestAssuredAPIChaining.CreateUser.tes_createUser(CreateUser.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- tes_createUser -->
</testsuite> <!-- Test1 -->
